### DISCUSSION TITLE
When working on Renovate self hosted one , unable to fetch new dependecies

### DISCUSSION THREAD

**Discussion by rajat053117** [renovatebot/renovate#27760] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Github

### Please tell us more about your question or problem

I have a file having only one image `us-east1-docker.pkg.dev/xxx/xxxx/xxx-xxx:7.03.0-xx.falconxxxxx@sha256:xxxxx` , I am unable to fetch the latest version of this 

This is my json `  "customManagers": [
    {
      "customType": "regex",
      "fileMatch": [
        "docker_images_list.txt"
      ],
      "matchStrings": [
        '(?<depName>.*?):(?<currentValue>.*?)@(?<currentDigest>sha256:[a-f0-9]+)\\s'
      ],
      "datasourceTemplate": "docker"
    }
  ]
}` and this is my package rule `    {
      "matchDatasources": ["docker"],
      "matchPackageNames": ["xxxxx"],
      "versioning": "regex:^(?<major>\d+)(?:\.(?<minor>\d+|))?(?:\.(?<patch>\d+))?(?:-(?<releaseMajor>\d+)(?:\.(?<releaseMinor>[^.]+(?:\.[^.]+)*))?)?$"
    },`

### Logs (if relevant)

<details><summary>Logs</summary>

```

```

</details>

**Comment by rarkins** [comment]
Which exact version of renovate are you using? Ensure you retest against latest and confirm

**Reply by rajat053117** [reply]
Hello @rarkins  I am using renovate slim version

**Reply by rajat053117** [reply]
renovate-version: 37.149-slim

**Reply by rajat053117** [reply]
I think the issue is with regex versioning ? Can you help me on this ? This the image name us-east1-docker.pkg.dev/d-gs-secops-cp/falcon/falcon-sensor:7.03.0-15805.falcon-linux.x86_64.Release.US-2. and versioning needs to fetch the version as 7.03.0-15805.falcon-linux.x86_64.Release.US-2. 
I am using this `"versioning": "regex:^(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)(?:-(?<compatibility>.+))?(?:-(?<build>\d+))?$"`

**Reply by viceice** [reply]
remove the slim suffix from docker image, it's deprecated

**Reply by rarkins** [reply, marked as answer]
What happens when you have default versioning, or set versioning to "loose"?

**Reply by rajat053117** [reply]
Thanks its resolved by using versioning to loose
